Traceback (most recent call last):
  File "/usr/share/miniconda3/envs/jupyter-book-env/lib/python3.7/site-packages/jupyter_cache/executors/utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "/usr/share/miniconda3/envs/jupyter-book-env/lib/python3.7/site-packages/nbclient/client.py", line 1117, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/usr/share/miniconda3/envs/jupyter-book-env/lib/python3.7/site-packages/nbclient/util.py", line 78, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/usr/share/miniconda3/envs/jupyter-book-env/lib/python3.7/site-packages/nbclient/util.py", line 57, in just_run
    return loop.run_until_complete(coro)
  File "/usr/share/miniconda3/envs/jupyter-book-env/lib/python3.7/asyncio/base_events.py", line 587, in run_until_complete
    return future.result()
  File "/usr/share/miniconda3/envs/jupyter-book-env/lib/python3.7/site-packages/nbclient/client.py", line 559, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "/usr/share/miniconda3/envs/jupyter-book-env/lib/python3.7/site-packages/nbclient/client.py", line 862, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/usr/share/miniconda3/envs/jupyter-book-env/lib/python3.7/site-packages/nbclient/client.py", line 765, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import nibabel as nib
from nilearn.plotting import view_img

zmap = nib.load('../data/unthresh_Z.nii.gz')
view_img(zmap, threshold=2.8, figsize=(25,10))
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mModuleNotFoundError[0m                       Traceback (most recent call last)
[0;32m/tmp/ipykernel_4563/2598452455.py[0m in [0;36m<module>[0;34m[0m
[0;32m----> 1[0;31m [0;32mimport[0m [0mnibabel[0m [0;32mas[0m [0mnib[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      2[0m [0;32mfrom[0m [0mnilearn[0m[0;34m.[0m[0mplotting[0m [0;32mimport[0m [0mview_img[0m[0;34m[0m[0;34m[0m[0m
[1;32m      3[0m [0;34m[0m[0m
[1;32m      4[0m [0mzmap[0m [0;34m=[0m [0mnib[0m[0;34m.[0m[0mload[0m[0;34m([0m[0;34m'../data/unthresh_Z.nii.gz'[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m      5[0m [0mview_img[0m[0;34m([0m[0mzmap[0m[0;34m,[0m [0mthreshold[0m[0;34m=[0m[0;36m2.8[0m[0;34m,[0m [0mfigsize[0m[0;34m=[0m[0;34m([0m[0;36m25[0m[0;34m,[0m[0;36m10[0m[0;34m)[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m

[0;31mModuleNotFoundError[0m: No module named 'nibabel'
ModuleNotFoundError: No module named 'nibabel'

